packages/aws-cdk/lib/cli/user-configuration.ts

```diff
@@ -305,8 +305,9 @@ export function commandLineArgumentsToSettings(argv: Arguments): Settings {
     ignoreNoStacks: argv['ignore-no-stacks'],
     hotswap: {
       ecs: {
-        minimumEcsHealthyPercent: argv.minimumEcsHealthyPercent,
-        maximumEcsHealthyPercent: argv.maximumEcsHealthyPercent,
```

Comment:
This allows for passing the hotswap override properties via the command line. However, it never actually worked because the keys it sets are not actually the keys we read:

https://github.com/aws/aws-cdk-cli/blob/382393730b6e8870cfb5d22aaf40c51b0b73b887/packages/aws-cdk/lib/cli/cdk-toolkit.ts#L387-L393

- `minimumEcsHealthyPercent` !== `minimumHealthyPercent`
- `maximumEcsHealthyPercent` !== `maximumHealthyPercent`